zbus/tests/issue/issue_310.rs

```diff
@@ -19,7 +19,7 @@ async fn issue_310() {
 
     #[zbus::interface(name = "net.connman.iwd.Station")]
     impl Station {
-        #[zbus(property)]
+        #[zbus(property(emits_changed_signal = "true"))]
```

Comment:
This is the default, so not sure why this is needed.

Reply:
Indeed. Removed